testing/ssh-tools/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Contributor: Sven Wick <sven.wick@gmx.de>
+# Maintainer: Sven Wick <sven.wick@gmx.de>
+
+pkgname=ssh-tools
+pkgver=1.5
+pkgrel=0
+pkgdesc="collection of various tools using ssh"
+arch="noarch"
+url="https://github.com/vaporup/ssh-tools/"
+license="GPL-3.0"
+#depends="bash ncurses openssh-client colordiff"
+depends="bash cmd:ssh cmd:ssh-keygen cmd:ssh-keyscan cmd:tput"
+source="$pkgname-$pkgver.tar.gz::https://github.com/vaporup/${pkgname}/archive/v${pkgver}.tar.gz"
+options="!check"
```

Comment:
A comment about why checks are disabled is helpful.

Reply:
I followed the advise from abuild:

>  declare !check in $options to indicate the package does not have a testsuite

Would this change be OK ?

`options="!check"   # the package does not have a testsuite`